src/components/molecules/Visualizer/Widget/index.tsx

```diff
@@ -73,8 +73,8 @@ export default function WidgetComponent<PP = any, SP = any>({
 
   if (!w) return null;
 
-  const Builtin =
-    w?.pluginId && w.extensionId ? builtin[`${w.pluginId}/${w.extensionId}`] : undefined;
+  const builtinWidgetId = `${w.pluginId}/${w.extensionId}`;
+  const Builtin = isBuiltinWidget(builtinWidgetId) ? builtin[builtinWidgetId] : undefined;
```

Comment:
Please tell me if you think I'm wrong (because I'm not overly confident in dealing with Records, but are we not overly complicating this (both the old code and the new code)?
Can't we just do
```
  const Builtin = builtin[`${w.pluginId}/${w.extensionId}`];
```
and then it'll either return a builtin component to render below or be undefined?

Reply:
The reason why I'm using `isBuiltinWidget()`  means that I want to assert `builtinWidgetId` as `builtin` object's key. If we don't assert `builtintWidgetId` then we can't access `builtin` object without `as`  keyword like the following.

```
builtin[builtinWidgetId as keyof typeof builtin];
```

But this is not reliable...